src/frame/pluginmanager.cpp

```diff
@@ -31,6 +31,9 @@ using namespace DCC_NAMESPACE;
 
 const static QString TranslateReadDir = QStringLiteral(TRANSLATE_READ_DIR);
 
+// The punctuation symbol used to separate items in a list. e.g. A, B, C, D")
```

Comment:
```suggestion
// The punctuation symbol used to separate items in a list. e.g. A, B, C, D"
```

ps. 译者是看不到代码注释的，双参数的 tr("") 的目的就是确保消歧义提示是译者看得到的。

Reply:
done